src/store.ts

```diff
@@ -326,17 +332,21 @@ export const initializeFirestoreListeners = (onLoad: () => void): (() => void) =
       const plan = getFirstPlan(data);
       store.commit('setPlans', data.plans);
       store.commit('setCurrentPlan', plan);
+      store.commit('setSavedCourses', data.savedCourses); // Note: toggle this on and off to save collections progress after refresh
       const { orderByNewest } = data;
       store.commit('setSemesters', plan.semesters);
       updateDoc(doc(fb.semestersCollection, simplifiedUser.email), {
         plans: data.plans,
+        // savedCourses: data.savedCourses,
```

Comment:
I saw the toggling comment from above... is there a more intuitive way (rather than toggling within the code) to handle the refresh progress? works for now, but something to consider.